packages/query/src/user/useNavigatorOnline.ts

```
@@ -1,17 +1,18 @@
-import React from "react"
+import React, { useCallback } from "react"
 
 const getOnLineStatus = () =>
   typeof navigator !== "undefined" && typeof navigator.onLine === "boolean"
     ? navigator.onLine
     : true
 
-const useNavigatorOnLine = () => {
+const useNavigatorOnLine = (): Boolean => {
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)